packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:async';
 import 'dart:math' as math;
 
+import 'package:flutter/material.dart';
```

Comment:
You shouldn't import material from within material. Use a relative import instead.